roles/build-release/tasks/tests.yaml

```diff
@@ -22,44 +22,29 @@
 
 # Note: the version of ansible-core doesn't necessarily match the deps file since the version requirement is >=
 - block:
-    # Note: the value is either _ansible_base_version or _ansible_core_version depending on the version
-    # ex: https://github.com/ansible-community/ansible-build-data/blob/main/4/ansible-4.4.0.deps
-    - name: Retrieve the expected minimum version of ansible-core
-      shell: >-
-        grep -E "_ansible_(base|core)_version" {{ antsibull_data_dir }}/{{ _deps_file }} | awk '{print $2}'
-      changed_when: false
-      register: _expected_ansible_core
+    - name: Include deps_file
+      ansible.builtin.include_vars:
+        file: "{{ antsibull_data_dir }}/{{ _deps_file }}"
+        name: _deps
 
-    - name: Retrieve the installed version of ansible-core
-      shell: >-
-        {{ antsibull_ansible_venv }}/bin/pip show ansible-core | awk '/Version/ {print $2}'
-      changed_when: false
-      register: _installed_ansible_core
+    - name: Query installed pip packages
+      community.general.pip_package_info:
```

Comment:
community.general isn't available, the role only assumes ansible-core (and that's what's installed in CI).

Reply:
Sure. That would be a breaking change though. Since we're still in pre 1.0.0 releases that's not a big deal though.